pod/perldelta.pod

```diff
@@ -226,13 +226,65 @@ XXX L<message|perldiag/"message">
 
 =head2 Changes to Existing Diagnostics
 
-XXX Changes (i.e. rewording) of diagnostic messages go here
-
 =over 4
 
 =item *
 
-XXX Describe change here
+Error messages that output class or package names have been modified to
+output double quoted strings with non-word characters escaped. If a
```

Comment:
Perhaps you mean "non-printable"? Numbers and other punctuation are usually considered "non-word" but they're still handled literally:

```
$ bleadperl -e '"a+b0c\nd"->new'
Can't locate object method "new" via package "a+b0c\nd" (perhaps you forgot to load "a+b0c\nd"?) at -e line 1.
```

Reply:
Actually digits and underbars are word chars in perl:

```
$ perl -le'print "1"=~/\w/ ? "word" : "not word"'
word
```

But yes, its a good point. I will rephrase it. The logic is actually a bit more complicated. If the codepoint is ASCII then it will be escaped if it is non printable or a double quote or slash, and if it is larger than 255 it will be escaped if it is not a wordchar. 

Not exactly the easiest thing to sum up. We dont want to make error messages unreadable for people working with unicode package names in other languages, and in theory identifiers should match IDFIRST IDREST+, which basically means "word characters", along of course with the colon, which we don't want to escape. Anyway, I am not certain that preciseness here is necessary, ill figure out a different wording.

Reply:
Anyway, new wording pushed.